### PR TITLE
[Scheduler] Add `AsCronTask` and `AsPeriodicTask`

### LINKED ISSUE #18949: [Messenger][Scheduler] Add AsCronTask & AsPeriodicTask attributes
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#51525
| PR author(s) | @valtzu
| Merged in    | 6.4

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 6.4 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix #18949

### CODE REVIEW

File: scheduler.rst [hunk 1]
```diff
@@ -211,6 +215,34 @@ For example::
...
+    #[AsCronTask('0 0 * * *')]
+    class SendDailySalesReports
+    {
+        public function __invoke()
+        {
+            // ...
+        }
+    }
+
+If you want to use another method name than ``__invoke()``, you can specify it
```
Comment (alamirault) [inline scheduler.rst]: I think you should also document `timezone`, `jitter`, `arguments`, `schedule`

File: scheduler.rst [hunk 2]
```diff
@@ -226,6 +258,32 @@ defined by PHP datetime functions::
...
+Like cron triggers, you can also use the
+:class:`Symfony\\Component\\Scheduler\\Attribute\\AsPeriodicTask` attribute
+on an invokable class::
+
+    // src/Scheduler/Task/SendDailySalesReports.php
+    namespace App\Scheduler\Task;
+
+    use Symfony\Component\Scheduler\Attribute\AsPeriodicTask;
+
+    #[AsPeriodicTask(frequency: '1 day', from: '2022-01-01', until: '2023-06-12')]
```
Comment (alamirault) [inline scheduler.rst:333]: Should we talk about ? 

`frequency` can also be an int (from my understand it's for seconds)
`from` can be nullable
`until` can be nullable
`jitter`
`schedule`

File: scheduler.rst (hunk 1 above)
Comment (alexandre-daubois) [reply scheduler.rst]: Added some examples 🙂

File: scheduler.rst (hunk 2 above)
Comment (alexandre-daubois) [reply scheduler.rst:333]: Added some other as well!

**Comment by javiereguiluz** [comment]
We've just merged #19292 (from Dec. 2023) that adds many things, including these two attributes.

I'm going to merge this PR too and try to combine some ideas from it in the existing docs. Thanks.

**Comment by javiereguiluz** [comment]
Merged! Thanks a lot Alex.

**Pull request merged**